src/style.css

```diff
@@ -1 +1,107 @@
-/* add styles here */
+@font-face{
+  font-family: Avenir;
+  src:url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face{
+  font-family: Avenir;
+  src:url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+:root {
+  --basic-color: #3d4e61;
+}
+
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
+
+input {
+  font-family: inherit;
```

Comment:
- Don't use tag names for styling (except html and body)
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/63243136/b5dc7eda-46cc-498a-8915-a32d09b90f4b)